opencv/src/OpenCV/Core/ArrayOps.hs

```diff
@@ -387,6 +388,29 @@ matScalarCompare src x cmpType = unsafeWrapException $ do
     c'x = realToFrac x
     c'cmpOp = marshalCmpType cmpType
 
+-- | Performs a per-element comparison of two matrices.
+matCompare
+    :: Mat shape ('S 1) depth
+    -> Mat shape ('S 1) depth
+    -> CmpType
+    -> CvExcept (Mat shape ('S 1) depth)
```

Comment:
IIUC the [OpenCV documentation suggests](http://lm-bletchley.lumi.guide/store/key/42-dcri803/revision-infos) that the output depth should be `Word8`.

Reply:
Good point! Fixed :small_airplane: